articles/email/custom.md

```diff
@@ -49,22 +51,19 @@ function (user, context, callback) {
 }
 ```
 
-After calling the API you'll [add a flag to the user's profile](https://auth0.com/docs/rules/metadata-in-rules) which specifies that the verification email has already been sent to make sure the email is only sent once.
+### Custom redirect
 
-Since the email will be sent by your own API you now have complete control over the verification email. The only thing you need to keep in mind is that the users still need to click the verification link to verify their email address.
+A custom redirect is useful when you want to direct users to certain URLs based on user attributes or on the tenant.
 
-The Auth0 API exposes [the verification endpoint](https://auth0.com/docs/api/v2#!/Tickets/post_email_verification) which makes it possible to generate the email verification link for a specific user. The endpoint also allows you to specify the `resultUrl` which is the URL to which your users will be sent after they validated their email address.
+The Auth0 API exposes a [verification endpoint](/api/v2#!/Tickets/post_email_verification) that generates the email verification link for each user. This endpoint allows you to specify the `resultUrl` to which users will be redirected after they have validated their email address by clicking the link in the verification email.
 
-It's also possible to use the Auth0 API to [send verification emails directly](https://auth0.com/docs/api/v2#!/Jobs/post_verification_email), but this will always redirect users to the same URL after clicking on the verification link.
-This URL is configurable [in the dashboard](${uiURL}/#/emails).
-
-As a result you now also have complete control over the URL the users will be redirected to after validation. This can be useful when users need to be redirected to specific pages based on their account, on their subscription, on the tenant, ...
+**NOTE:** You can call the [send a verifiy email address email](/api/v2#!/Jobs/post_verification_email) endpoint to send the verification email directly. However, this endpoint will redirect all users to the same URL which is specified in the `Redirect To` field under the **Verification Email** tab on the [Email Templates](${uiURL}/#/emails) page of the Auth0 dashboard.
```

Comment:
Yeah, this is bad UX. But you can set the field then disable the email, but the field is still valid. You may want to make this a warning or something. I would say that the user shouldn't use our post_verification_email API if they are doing it custom. Maybe you want to say, warning, the post_verfication_email endpoint will still work, and will send them to the redirect... ... but you will likely not want to use it... because you have your own implimentation.